Gemfile.lock

```diff
@@ -139,9 +141,9 @@ GEM
       oauth2 (~> 1.1)
       omniauth (>= 1.1.1)
       omniauth-oauth2 (>= 1.6)
-    omniauth-oauth2 (1.7.0)
+    omniauth-oauth2 (1.7.1)
       oauth2 (~> 1.4)
-      omniauth (~> 1.9)
```

Comment:
You may want to double-check that we're not pushing up this. I think we want to keep it as `omniauth (~> 1.9)`

Reply:
I believe it was an issue when oauth was version 2.0 please correct me if I am wrong but all test passed